python/ember/_ember.pxd

```diff
@@ -48,6 +48,7 @@ cdef extern from "readConfig.h":
         cbool fixedBurnedVal, fixedLeftLoc, twinFlame
         int continuityBC
         cbool curvedFlame, unburnedLeft, fuelLeft
+        cbool axiJetFlame
```

Comment:
Is there a better name for this configuration (or both configurations), given that both of the configurations in cylindrical coordinates are axisymmetric. Maybe we should think of this as the 'disc' flame, and the other curved configuration as a 'cylindrical' flame?

Reply:
Fair point. I refactored both variables and then added a unified 'flameGeometry' option under general options to allow the user to specify 'planar', 'cylindrical', or 'disc.' The default is 'planar.'